cfgov/legacy/views/complaint.py

```diff
@@ -24,3 +24,22 @@ def get_context_data(self, **kwargs):
         })
 
         return context
+
+    def dispatch(self, *args, **kwargs):
+        response = super(ComplaintLandingView, self).dispatch(*args, **kwargs)
+        response['Edge-Cache-Tag'] = 'complaints'
+        return response
+
+
+class CCDBSearchView(TemplateView):
+    """Consumer Complaint Database search page view.
+
+    This view renders the template for the CCDB search application page.
+    """
+
+    def dispatch(self, *args, **kwargs):
+        response = super(CCDB5SearchView, self).dispatch(*args, **kwargs)
```

Comment:
Sorry – must have copied at the wrong time.
```suggestion
        response = super(CCDBSearchView, self).dispatch(*args, **kwargs)
```